Source/WebKit/WebProcess/WebPage/WebFrame.cpp

```diff
@@ -592,7 +592,7 @@ void WebFrame::didReceivePolicyDecision(uint64_t listenerID, PolicyDecision&& po
     }
 
     if (policyDecision.policyAction == PolicyAction::Use && policyDecision.sandboxExtensionHandle) {
-        if (auto* page = this->page()) {
+        if (RefPtr page = this->page()) {
```

Comment:
`page` doesn't need `RefPtr` here, since all functions called on `page` are trivial and inlined.